package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "lint": "eslint .",
```

Comment:
you should probably run
`npm install eslint --save-dev`
and checkin the package.json

Reply:
The dev dependency is already there, it was part of the template I used to create the project.